ui/dashboard/src/components/ellipsis-spinner/index.tsx

```diff
@@ -0,0 +1,14 @@
+import styles from './styles.module.css';
+
+const EllipsisSpinner = () => {
+  return (
+    <div className={styles.main}>
+      <div></div>
+      <div></div>
+      <div></div>
+      <div></div>
```

Comment:
What is this for?

Reply:
These elements create the loading. Do you like the loading like this? or can we only use spinner loading?

https://github.com/user-attachments/assets/2fc8a2b0-8a03-43c5-a283-93d101ff9ec8

Reply:
https://github.com/user-attachments/assets/bc9c532e-a93b-4ecd-abe3-68c7a930e040

Reply:
@steveninhle, the second one, please.